crossbeam/experiment/train_eval.py

```diff
@@ -191,7 +191,8 @@ def _gather_eval_info(rank, device, local_acc, local_num):
 
 
 def train_eval_loop(args, device, model, train_files, eval_tasks,
-                    task_gen, trace_gen):
+                    task_gen, trace_gen, checkpoint):
+  random.shuffle(train_files)
```

Comment:
I think we shouldn't shuffle train_files so that training runs are more consistent/reproducible